packages/catalog-outbound-writer/src/converters/toOutboundEventV2.ts

```diff
@@ -27,6 +27,7 @@ function toOutboundDescriptorV2(
 ): Exact<OutboundEServiceDescriptorV2, EServiceDescriptorV2> {
   return {
     ...descriptor,
+    templateVersionRef: undefined, // todo outbound library must be updated
```

Comment:
The outbound library should be updated before merging this PR or after?

Reply:
After the release on DEV

Reply:
Yes, there is a dedicated story